engine/internal/runtime/manager.go

```diff
@@ -291,5 +305,8 @@ func (m *Manager) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&appsv1.StatefulSet{}, builder.WithPredicates(filterByAnno)).
 		WithLogConstructor(constructer).
+		// To share the runtime deletion event, disable the leader election
+		// for this controller if the processor disables it.
```

Comment:
Slightly prefer "if the processor disables the leader election" to clarify what "it" refers to.